node/src/main/scala/coop/rchain/node/runtime/NodeRuntime.scala

```diff
@@ -158,22 +156,6 @@ class NodeRuntime[F[_]: Monixable: ConcurrentEffect: Parallel: Timer: ContextShi
         */
       _ <- mkDirs(dataDir)
 
-      dagConfig = BlockDagFileStorage.Config(
```

Comment:
Additional commit is to apply changes from https://github.com/rchain/rchain/pull/3363 due to NodeRuntime has been split and moved to different package in block-merge branch